wgengine/magicsock/peermtu.go

```diff
@@ -110,3 +115,10 @@ func (c *Conn) UpdatePMTUD() {
 	c.peerMTUEnabled.Store(newStatus)
 	c.resetEndpointStates()
 }
+
+func isPMTUError(err error) bool {
+	if errors.Is(err, unix.EMSGSIZE) {
```

Comment:
can simplify to return `errors.Is(...`

looks like we have an existing pattern for unwrapping these types of errors, take a look at `endpoint_default.go` and the stub `endpoint_stub.go`